source/frontend/src/mocks/compensations.mock.ts

```diff
@@ -93,7 +93,7 @@ export const getMockApiCompensation = (): Api_CompensationRequisition => ({
       compensationRequisition: null,
       isPaymentInTrust: true,
       gstNumber: '9999',
-      acquisitionOwnerId: null,
+      acquisitionOwnerId: 1,
```

Comment:
Is this relevant to this PR? I didn't see this property used in this PR

Reply:
I wouldn't let me submit the form otherwise. It needed at least one payee in the form.